src/components/sharing/index.js

```diff
@@ -209,7 +209,7 @@ function Sharing(props) {
                     sharing.forEach((shares) => {
                         const updates = getShareResponseValues(shares);
                         const successFalse = updates.filter(
-                            (update) => update.success === false
```

Comment:
In this case the response bodies from other sharing and unsharing endpoints are also being processed, so we still need to examine the response bodies. For the analysis sharing and unsharing endpoints, the `success` flag will not exist. (The analysis sharing and unsharing endpoints use an `ok` flag instead because they're now asynchronous, and it's impossible to predict with complete accuracy whether or not the analysis will be shared successfully.) This means that an analysis sharing or unsharing response will never match the response body scanning code here. This is okay, however, because the analysis sharing and unsharing endpoints will only return an error in the response body if a preliminary validation check fails. And if that happens then the endpoint will return a 400 status. The response bodies from other sharing and unsharing endpoints will still be processed as they were before.